desktop/cmp/input/Select.js

```diff
@@ -547,3 +553,7 @@ export class Select extends HoistInput {
 
 }
 export const select = elemFactory(Select);
+
```

Comment:
At the risk of contradicting myself, vis a  vis cost of components, I will say that I would just inline this particular framework component above. If not, at least minimize by removing its model and observability.  

This is a framework component, we will have many of them in the world, and might as well keep it tight!  Also, should be able to get a better display name from the Icon component itself over a new anonymous component (although we may have to fix that)